README.md

```diff
@@ -21,38 +21,57 @@ When asked to remove the existing VCS, answer `Y`.
 composer require --dev zero-config/preacher
 ```
 
-Optionally, if one wants to always generate output when a file is committed, try
-the following:
+This installs the necessary files and packages. The next step is initiating a 
+git repository, so Preacher can keep track of what needs to be published:
 
 ```bash
-ln -s ../../vendor/bin/preach .git/hooks/post-commit
+git init
 ```
 
-This will install preacher as a post-commit hook and makes it run each time you
-commit one of your files.
+Now all we need to do, is commit our first page and template:
 
-# Custom templates
+```bash
+git add index.md default.html.twig  && git commit -m "My first Preacher page!"
+```
+To see what a basic .md page file and template can contain, have a look at Preacher's 
+own [index.md](https://raw.githubusercontent.com/ZeroConfig/Preacher/master/index.md) page
+and [default.html.twig](https://github.com/ZeroConfig/Preacher/blob/master/default.html.twig)
+template.
 
-By default, Preacher will look for the Twig template called `default.html.twig`.
-However, if a custom template is required, simply give it the same name as the
-source file.
 
-E.g.: `articles/something-fancy.md` => `articles/something-fancy.html.twig`.
+# Generating pages
+Preacher creates pages from the committed .md page files and .twig templates. It uses a 
+single command to either generate the current directory and all its children, 
+or one can (re)generate a single source file.
 
-# Usage
+```bash
+vendor/bin/preach [<source>]...
+```
 
-It uses a single command to either generate the current directory and all its
-children or one can (re)generate a single source file.
+Optionally, if one wants to always generate output when a file is committed, try
+the following:
 
 ```bash
-vendor/bin/preach [<source>]...
+ln -s vendor/bin/preach .git/hooks/post-commit
```

Comment:
Please restore this to `../../vendor/bin/preach`, as this version was a bug. It's because of how symlinks work. Relative means relative to the target, which puts the working directory at `.git/hooks/` when the target is being made. Therefore, `../../vendor/bin/preach` points to the correct location. I always fall for this myself, hence why it was broken in the first place.